test/common/tests.go

```diff
@@ -91,6 +91,7 @@ var (
 				// Keep H11 as last 3scale IDP Test as test causes 3scale deployments to be rescaled at the end of test
 				// Can potentially cause subsequent tests be flaky due to waiting for 3scale deployments to complete
 				{"H11 - Verify 3scale SMTP config", Test3ScaleSMTPConfig},
+				{Description: "H24 - Verify selfmanaged Apicast", Test: TestSelfmanagedApicast},
```

Comment:
Could we move this test to **before** H11 ? We want to keep H11 as the last IDP test as mentioned to the code comments :thinking:

Reply:
Done. Sorry, didn't pay attention to comments